### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by shajoezhu**
close https://github.com/insightsengineering/random.cdisc.data/issues/311

### CODE REVIEW

**Comment by shajoezhu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pawelru** [comment]
Please have a look at this: https://github.com/insightsengineering/coredev-tasks/issues/577#issuecomment-2295842071

File: DESCRIPTION [hunk 1]
```diff
@@ -51,7 +50,7 @@ Config/Needs/verdepcheck: mllg/checkmate, tidyverse/dplyr,
     tidyverse/stringr, tidyverse/tibble, tidyverse/tidyr,
     yaml=vubiostat/r-yaml, gowerc/diffdf, yihui/knitr, rstudio/rmarkdown,
     r-lib/testthat, r-lib/withr
-Config/Needs/website: insightsengineering/nesttemplate
+Config/Needs/website: markdown, insightsengineering/nesttemplate
```
Comment (Melkiades) [inline DESCRIPTION:53]:
```suggestion
Config/Needs/website: rstudio/rmarkdown, insightsengineering/nesttemplate
```

File: DESCRIPTION [hunk 2]
```diff
@@ -39,19 +39,19 @@ Imports:
 Suggests:
     diffdf,
     knitr (>= 1.42),
-    rmarkdown (>= 2.23),
     testthat (>= 3.0.4),
     withr (>= 2.0.0)
 VignetteBuilder:
-    knitr
+    knitr,
+    rmarkdown (>= 2.23)
```
Comment (shajoezhu) [inline DESCRIPTION:46]:
```suggestion
    rmarkdown
```

**Review by Melkiades — APPROVED** [review]
Lgtm! :)

File: DESCRIPTION [hunk 3]
```diff
@@ -43,15 +43,16 @@ Suggests:
...
+    rmarkdown
 RdMacros:
     lifecycle
 Config/Needs/verdepcheck: mllg/checkmate, tidyverse/dplyr,
     r-lib/lifecycle, tidyverse/lubridate, tidyverse/magrittr, r-lib/rlang,
     tidyverse/stringr, tidyverse/tibble, tidyverse/tidyr,
     yaml=vubiostat/r-yaml, gowerc/diffdf, yihui/knitr, rstudio/rmarkdown,
     r-lib/testthat, r-lib/withr
-Config/Needs/website: insightsengineering/nesttemplate
+Config/Needs/website: rstudio/rmarkdown, insightsengineering/nesttemplate
```
Comment (pawelru) [inline DESCRIPTION:55]: hey @shajoezhu @Melkiades 
this one is redundant - can you revert it please?

Comment (Melkiades) [reply DESCRIPTION:55]: yes will do!

**Pull request merged**